plugins/corsair/fu-plugin-corsair.c

```diff
@@ -21,6 +21,7 @@ fu_plugin_corsair_load(FuContext *ctx)
 {
 	fu_context_add_quirk_key(ctx, "CorsairDeviceKind");
 	fu_context_add_quirk_key(ctx, "CorsairVendorInterfaceId");
+	fu_context_add_quirk_key(ctx, "CorsairSubdeviceId");
```

Comment:
This needs documenting in README.md -- and I'm a bit confused what it actually is.